aiobotocore/credentials.py

```diff
@@ -364,7 +362,9 @@ async def get_frozen_credentials(self):
         return self._frozen_credentials
 
 
-class AioDeferredRefreshableCredentials(AioRefreshableCredentials):
+class AioDeferredRefreshableCredentials(
+    DeferredRefreshableCredentials, AioRefreshableCredentials
+):
```

Comment:
little value in inherting from parent class as effictively it's a re-write bug 🤷🏼  i guess useful in case they do an isinstance/issubclass on it

Reply:
In ned to differ: by subclassing, we avoid having to copy over a bunch of non-blocking code from upstream.

Reply:
Well, in this case "a bunch" is just the `refresh_needed()` below, but still.

Reply:
I wonder if I left the parent out due to the parent having the method implemented and slicing or something 🤔

Reply:
I wondered, too, but couldn't find anything. It's really just copy and paste, but please double check!